rxandroid/src/main/java/rx/android/schedulers/HandlerScheduler.java

```diff
@@ -24,30 +24,32 @@
 import rx.subscriptions.Subscriptions;
 import android.os.Handler;
 
-class HandlerThreadScheduler extends Scheduler {
+/** A {@link Scheduler} backed by a {@link Handler}. */
+public final class HandlerScheduler extends Scheduler {
+    /** Creates a {@link Scheduler} which uses {@code handler} to execute actions. */
+    public static HandlerScheduler create(Handler handler) {
```

Comment:
What about `from(handler)` instead of `create(handler)` for consistency with `Schedulers.from(executor)`?

Also, users may expect that `AndroidSchedulers` will provide method `create(handler)`, because `Handler` is Android specific thing and `mainThread()` (which is based on `Handler`) lives in `AndroidSchedulers`, as far as I see, all RxJava's schedulers provided via single class — `Schedulers` so for consistency we can provide all RxAndroid's schedulers via `AndroidSchedulers`. 

Maybe we can move `create(handler)` to `AndroidSchedulers` or move `mainThread()` to `HandlerScheduler` for more obvious code structure?